src/NSubstitute/Core/ThreadLocalContext.cs

```diff
@@ -41,16 +41,21 @@ public void SetLastCallRouter(ICallRouter callRouter)
         public ConfiguredCall LastCallShouldReturn(IReturn value, MatchArgs matchArgs)
         {
             var lastCallRouter = _lastCallRouter.Value;
-            if (lastCallRouter == null) throw new CouldNotSetReturnDueToNoLastCallException();
-            if (!lastCallRouter.IsLastCallInfoPresent()) throw new CouldNotSetReturnDueToMissingInfoAboutLastCallException();
+            if (lastCallRouter == null)
```

Comment:
Don't worry about changing it here, but for future changes I'd prefer to put braces around any `if` blocks that span multiple lines (I'm happy to omit them when they're on a single line as a guard).

e.g.

```
// Single line
if (condition) someCall();
// Multi line
if (condition) 
{
  someCall();
}

// Error prone:
if (condition)
   someCall();
   anotherCallThatLooksLikeItIsPartOfIfButIsNotDueToIndentingFail();
```

Happy to be convinced otherwise. :)

Aside: I actually prefer:
```
if (condition) {
  someCall();
}
```

But unfortunately that doesn't seem standard for C# :(

Reply:
@dtchepak Roger that! 😅